packages/dd-trace/src/service-naming/schemas/v1/storage.js

```diff
@@ -14,7 +14,11 @@ const storage = {
       opName: () => 'memcached.command',
       serviceName: configWithFallback
     },
-    redis: redisNaming
+    redis: redisNaming,
+    tedious: {
+      opName: () => 'sqlserver.query',
```

Comment:
it should be `mssql.query` (sticking to OTEL conventions here: https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/database/)